include/deal.II/lac/la_parallel_block_vector.h

```diff
@@ -491,16 +491,19 @@ namespace LinearAlgebra
 
       /**
        * Set each block of this vector as follows:
-       * $U^i = \sum_{j} V_j A^{ji}$ where $U^i$
+       * $U^i = \sum_{j} V_j A^{ji} b$ where $U^i$
        * and $V_j$ indicate the $i$th block (not element) of $U$ and the
-       * $j$th block of $V$, respectively.
+       * $j$th block of $V$, respectively. If @p adding is <code>true</code>,
+       * the result will be added instead.
```

Comment:
> that is not explicitly documented, by the way

it **is** documented, you just reviewed the older version of this page, it seems.

Reply:
> so is there anything that speaks against scaling the matrix before entering this function

generally only convenience in case the FullMatrix does is not needed in "multiplied by -1" state.

> dgemm interface, i.e., introduce a parameter to scale the this vector and a second one to scale V_j with.

I am fine with that approach

Reply:
OK, the I would vote for adding a parameter `s` that scales the calling vector (defaulting to 0) and the parameter `b` for the input. I think that's both general and still something people recognize.